ballerina/tests/subscription_initiation_test.bal

```diff
@@ -26,15 +26,15 @@ const string HUB_FAILURE_URL = "http://127.0.0.1:9192/common/failed";
 const string COMMON_TOPIC = "https://sample.topic.com";
 
 service /common on new http:Listener(9192) {
-    isolated resource function get discovery(http:Caller caller, http:Request request) {
+    isolated resource function get discovery(http:Caller caller, http:Request request) returns error? {
         http:Response response = new;
         response.addHeader("Link", "<http://127.0.0.1:9192/common/hub>; rel=\"hub\"");
         response.addHeader("Link", "<https://sample.topic.com>; rel=\"self\"");
-        http:ListenerError? resp = caller->respond(response);
+        check caller->respond(response);
```

Comment:
Also you can return

Reply:
Will do this syntax update in another PR.